src/components/Game/index.js

```diff
@@ -25,20 +29,18 @@ class Game extends Component {
         <h2>score {this.state.score}</h2>
 
         {
-          (this.vocabulariesMap = vocabularies.map(
-            ({ name , image }) => (
-              <button class="imageButton">
-                <a
-                  href="#"
-                  onClick={() => {
-                    this.changeWord(vocabularies);
-                  }}
-                >
-                  <Card class="images" image={image} name={name} />
-                </a>
-              </button>
-            )
-          ))
+          (this.vocabulariesMap = vocabularies.map(({ name, image }) => (
+            <button class="imageButton">
+              <a
+                href="#"
+                onClick={() => {
+                  this.changeWord(vocabularies, name);
```

Comment:
This isn't the user answer, it's the name you get from the map. This needs to come from card they clicked.